README.md

```diff
@@ -65,34 +65,30 @@ TODO
 
 
 ### Elasticsearch
-Elasticsearch is automatically setup docker for development. To rebuild the search index from scratch,
-we run the following command in `docker-entrypoint.sh`:
-```sh
-python manage.py update_index
-```
+
 The search index is automatically updated using Wagtail's provided search signal handlers; but in case
 the index has to be rebuilt manually, run the following command:
-```sh
-docker-compose exec web python manage.py update_index
+```shell
+make update_index
 ```
 
 ## Handling Data
 
 ### HURUmap Data
 
 To import HURUmap data, we run the following command in `docker-entrypoint.sh`:
-```sh
+```shell
```

Comment:
`docker-entrypoint.sh` has been removed.

An aside: Since this is something that might be done a few times, shouldn't this be included in the Makefile e.g. `make pushdata`? Of course, it must read host and password from environmental variables (Same for HURUmap-Apps)

Reply:
Yes, in this repo as it's already in `hurumap`.